ee/ucp/admin/configure/use-your-own-tls-certificates.md

```diff
@@ -30,33 +30,27 @@ will have to download new ones to [access UCP from the CLI](../../user-access/cl
 
 ## Configure UCP to use your own TLS certificates and keys
 
-In the UCP web UI, log in with administrator credentials and
-navigate to the **Admin Settings** page.
+To configure UCP to use your own TLS certificates and keys:
 
-  In the left pane, click **Certificates**.
+1. In the UCP web UI, log in with administrator credentials and
```

Comment:
Change the first sentence to read, "Log into the UCP web UI with administrator credentials and navigate to the **Admin Settings** page.